src/main/java/org/sagebionetworks/web/client/widget/table/v2/TableEntityWidget.java

```diff
@@ -408,7 +410,7 @@ private void showSimpleSearchUI() {
 		actionMenu.setActionVisible(Action.SHOW_SIMPLE_SEARCH, false);
 		queryInputWidget.setShowSimpleSearchButtonVisible(false);
 		queryResultsWidget.setFacetsVisible(true);
-		actionMenu.setActionVisible(Action.SHOW_QUERY, true);
+		actionMenu.setActionVisible(Action.SHOW_QUERY, false);

```

Comment:
Is there a case where we're ever showing this button, or is it always supposed to be hidden now? If we never show it, let's remove the Action.SHOW_QUERY